polytope/polytope.py

```diff
@@ -442,6 +468,134 @@ def text(self, txt, ax=None, color='black'):
         _plot_text(self, txt, ax, color)
 
 
+def _translate(polyreg, d):
+    """Translate a polyreg by a vector in place. Does not return a copy.
```

Comment:
"Modified C{polyreg} in place" provides more information, and emphasizes that the object will change. Both ["in-place"](https://docs.python.org/3/howto/sorting.html) and ["in place"](https://docs.python.org/devguide/documenting.html) appear in Python's own documentation.